packages/atoms/src/components/kiwi-sortable-label/kiwi-sortable-label.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  Component,
+  ComponentInterface,
+  Event,
+  EventEmitter,
+  h,
+  JSX,
+  Prop,
+} from '@stencil/core';
+
+export type SortDirection = 'asc' | 'desc' | 'off';
+
+@Component({
+  tag: 'kiwi-sortable-label',
+  styleUrl: 'kiwi-sortable-label.css',
+  shadow: false,
+})
+export class KiwiSortableLabel implements ComponentInterface {
+  /**
+   * The label to be displayed.
+   */
+  @Prop()
+  label?: string;
+
+  /**
+   * The key to be sent by sort event.
+   */
+  @Prop()
+  sortKey!: string;
+
+  /**
+   * The initial sort direction.
+   */
+  @Prop()
+  sortDirection: SortDirection = 'off';
+
+  /**
+   * Sort event sent when clicked on a label.
+   */
+  @Event()
+  sort!: EventEmitter<string>;
+
+  render(): JSX.Element {
+    const sortIcon =
+      this.sortDirection === 'desc'
+        ? 'glyphicon-sort-by-alphabet-alt'
+        : 'glyphicon-sort-by-alphabet';
```

Comment:
Hier haben wir noch einen kleinen Bug der mir neulich aufgefallen ist:

```suggestion
    const sortIcon = {
      off: 'glyphicon-sort',
      desc: 'glyphicon-sort-by-alphabet-alt',
      asc: 'glyphicon-sort-by-alphabet',
    }[this.sortDirection];
```